Migration.md

```diff
@@ -27,6 +13,13 @@ release will remove the deprecated code.
    header files, the `ignition::` namespaces, and `IGN_*` and `IGNITION_*`
    macros.
 
+1. Behavior change in `math::Color` clamping functionality. Previously when
+   any of the color r, g, or b values are larger than 1, they are assumed to
+   be in the range of [0, 255] and the values are divided by 255
+   by the `math::Color::Clamp` funcion. This is no longer the case. Users are
+   expected to specify floating point r, g, b values in the range of [0, 1].
+   Any values larger than 1 will be clamped to 1.
+
```

Comment:
```suggestion
2. `math::SphericalCoordinates::LOCAL` now gives ENU coordinates instead of WNU. Both  `math::SphericalCoordinates::LOCAL2` and  `math::SphericalCoordinates::LOCAL` have the same behaviour now with `math::SphericalCoordinates::LOCAL` being deprecated.
```

Reply:
thanks, added in [5dceccb](https://github.com/gazebosim/gz-math/pull/619/commits/5dceccb428d4021fe360260ff4d92feccf15a6ab)